packages/runtime/src/tutorial-runner.ts

```diff
@@ -71,6 +72,22 @@ export class TutorialRunner {
     private _stepController: StepsController,
   ) {}
 
+  /**
+   *
+   * @param commands Commands schema
+   */
+  setCommands(commands: CommandsSchema) {
+    const newCommands = new Commands(commands);
+    let anyChange = this._changeDetection(commands);
```

Comment:
Newline to separate const from let?

Reply:
Oh actually `let` wasn't needed! I changed it to a `const`.